JavaScript/tests/syntax_test_js.js

```diff
@@ -1146,15 +1147,14 @@ var Constructor = function() {
 // Tests to ensure the new keyword is highlighted properly even when the
 // following element is not an identifier
 var abc = new ABC(
-//         ^ meta.instance.constructor keyword.operator.word.new
-//               ^ meta.instance.constructor meta.function-call.constructor
-//               ^ - meta.instance.constructor meta.instance.constructor
+//        ^^^ keyword.operator.word.new
```

Comment:
Imo it would make sense to add a negative assertion for the `new` keyword and the constructor meta scope.

Reply:
Added.